.github/workflows/ci.yml

```diff
@@ -136,21 +136,66 @@ jobs:
         run: |
           pytest -v tests
 
-  # GitHub allows you to set checks as required before PRs can be merged, which is annoying to do
-  # for matrix jobs like the one above which have six actual jobs. Instead we have a summary job
-  # like this, which we mark as required.
-  # Copied in part from:
-  # https://github.com/orgs/community/discussions/26822#discussioncomment-3305794
-  ci-report-status:
```

Comment:
why is this being removed?

Reply:
This was in place to avoid having to manually specify the "run tests" job for each platform and Python version when creating branch protection rules, which I just ended up doing anyway making this unnecessary